dataServices.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
What is the reason for this?

Reply:
To pass ESLint

Reply:
Remove this line, and fix the issues to pass the ESLint warnings